### ISSUE TITLE
Twitter specific Tokenizer

### ISSUE THREAD

**Issue by JherezTaylor** [melvynator/ELK_twitter#5]
I think it would be better to use a battle tested tokenizer specific to Twitter. The one from Carnegie Mellon is good and I can vouch for it's effectiveness. http://www.cs.cmu.edu/~ark/TweetNLP/#pos

I will TRY to make a PR.

**Comment by omarsar** [comment]
You are right! The current Twitter tokenizer is not robust enough. I am guessing everyone will require a different tokenizer for the task at hand. So it is good to make a note of it, or try to emphasize on the README for in case anyone is looking to use this for research purposes.

**Comment by JherezTaylor** [comment]
Yea that's true, there would be several use cases. Maybe it be ideal to define a tokenizer API that has several functions to return the text for different use cases? I'll look at the structure and see if it's possible.

**Comment by melvynator** [comment]
I think it can be a good Idea to have it as an API. We can build a pool of API and like that if someone develop a nice API in the lab we can just add it to the pool, and people can choose the one they want to use.

**Comment by omarsar** [comment]
Wow, so many good ideas. That would be a great side project. It is like building a tokenizing service more or less. Let's take a note of it on the projects section.

**Comment by melvynator** [comment]
Just want to submit an alternative to the API, maybe we could just define out own filter in Logstash: https://www.elastic.co/guide/en/logstash/current/_how_to_write_a_logstash_filter_plugin.html
